### DISCUSSION TITLE
User Insights Dashboard

### DISCUSSION THREAD

**Discussion by etherworlds** [openfoodfacts/open-prices#571] [Ideas]
From the Open Food Facts Days in October 2024, we generated an interesting list of insights for potential inclusion in a User Insights Dashboard (details on Raphael O's Miro board + Pierre's Figma board).  This answered the question: what's in it for me to contribute to Open Prices / Open Food Facts?

Afterwards, we talked about a potential hackathon with students (or each other?) to visualize some/all of those potential insights for the dashboard. 

For some user-centered Dashboard next steps, we can consider:
- currently available datapoints for use in the dashboard (list)
- potential other / new datapoints for use in the dashboard (list)
- simple visualization session, hackathon, or both? 
- attendees: is there a school, event, and/or group on which we want to focus? (or more than 1?)
- virtual, in-person, hybrid?
- when?
- context / personas / challenge statement
- volunteer leads
- brief creation

**Comment by etherworlds** [comment]
Are all GDPR dumps and/or receipts automatically added to my profile? 
(I vote for a **checkbox** because I've begun collecting receipts at the checkout which other people leave behind 🤓

**Comment by etherworlds** [comment]
What kinds of functionality would we need for splitting out items on a receipt and divvying them up amongst family member items? 
(i.e. poptarts or sugary cereal for the kids, a spouse's favorite chips, etc)

**Comment by raphodn** [comment]
Recent changes linked to this discussion:
- when users add a RECEIPT, they are now asked if it should go in their "consumption" stats
  - GDPR & RECEIPT (with "owner_consumption" checkbox set to true) go in the CONSUMPTION kind
  - the rest (PRICE_TAG) go in the COMMUNITY kind
- this allows to split the user dashboard into 2 tabs: 
  - User consumption ; Other contributions
  - https://prices.openfoodfacts.org/dashboard

Issues to follow this recent progress:
- https://github.com/openfoodfacts/open-prices/issues/727 & https://github.com/openfoodfacts/open-prices/issues/865
- https://github.com/openfoodfacts/open-prices-frontend/issues/1461

cc @serpico @teolemon

**Comment by serpico** [comment]
I noticed that currently when user display, for example, the My consumption tab of his dashboard, a maximum of 25 product cards are loaded when we scroll down, then there's a _My Prices_ button at the bottom which loads more cards but regardless of the fact they come from consumption or community, the button points to https://prices.openfoodfacts.org/dashboard/prices which mixes both "types" of prices. 

probably should be on [#1461](https://github.com/openfoodfacts/open-prices-frontend/issues/1461) @raphodn you'll move it if you think it's appropriate.

**Reply by raphodn** [reply]
Indeed !! I need to add a "kind" filter on these price/proof lists, i checked the done box too quickly ^^

**Reply by raphodn** [reply]
done in the next hour or so :)

**Comment by Freso** [comment]
Idk if this topic is also for brainstorming potential user insights for the users? Because I’d love to be able to see a breakdown of my consumption across various categories. Maybe both per-proof, but also aggregating like per-month or per-year would be nice. Possibly even an over-time view to see trends/changes?

**Comment by tuukka** [comment]
Can I find links to the Figma and Miro boards mentioned?

What I could find was this related Figma board mentioned in the wiki: https://www.figma.com/design/nFMjewFAOa8c4ahtob7CAB/Mobile-App-Design-(Quentin)?node-id=5787-19012&t=UoETOgDYDten9odq-0
